### ISSUE TITLE
How do I remap to a different key besides ctrl-p

### ISSUE THREAD

**Issue by zarry** [JamshedVesuna/vim-markdown-preview#14]
In the documentation you mention the comment below.  When I checked the installation section I did not see anything about remapping the key.  Thanks in advance! 

```
When in a *.markdown or *.md file, vim-markdown-preview does the following when you type Ctrl-p (can be remapped, see Installation):
```

**Comment by JamshedVesuna** [comment]
Good eye! Currently, I don't have a variable to change this easily, you have to change he mapping on lines [108](https://github.com/JamshedVesuna/vim-markdown-preview/blob/master/plugin/vim-markdown-preview.vim#L108) and [112](https://github.com/JamshedVesuna/vim-markdown-preview/blob/master/plugin/vim-markdown-preview.vim#L112).

I'll add a cleaner global variable very soon.

**Comment by zarry** [comment]
I actually had issues running this, it complained of a bad variable on this line I believe?  I have since uninstalled it and went back to manually opening it in my browser with an extension.  

I can re-install and try again if you would like the exact error.  I would definitely prefer to use this.  I was attempting to use github flavored markdown but I don't think it had anything to do with that. I turned the feature off and received the same error.  

https://github.com/JamshedVesuna/vim-markdown-preview/blob/master/plugin/vim-markdown-preview.vim#L76

**Comment by JamshedVesuna** [comment]
Interesting. Do you have `vim_markdown_preview_github` set to 1? What other global variables are you enabling with this plugin? Are you using `grip` or `markdown` to render the markdown file?

It would be great to get the exact error and environment.

**Comment by zarry** [comment]
I threw a new issue up. 

https://github.com/JamshedVesuna/vim-markdown-preview/issues/15

**Comment by JamshedVesuna** [comment]
Closed by 9088e7d154419296dde1794e58037537b1d18c67. Would be great if there was a cleaner way to do this, open to suggestions.
